src/lib.rs

```diff
@@ -412,6 +412,23 @@ pub mod tests {
                 }
             }
         }};
+        ($left:expr, $right:expr, $epsilon:expr) => {{
+            match (&$left, &$right, &$epsilon) {
+                (left_val, right_val, epsilon_val) => {
+                    let diff = (left_val - right_val).abs();
+
+                    if diff > *epsilon_val {
```

Comment:
can the code above call this part with epsilon `0.0005`

Reply:
I'm not sure I got it. Are asking for a change of epsilon in extended_stats tests or for a change in the macro using a fixed value?

Reply:
the macro has two code paths, one with a parameter and one with epsilon `0.0005`. The `0.0005` path can call the  variant with the parameter